src/react/ErrorPage.jsx

```diff
@@ -28,12 +28,17 @@ class ErrorPage extends Component {
                 description="error message when an unexpected error occurs"
               />
             </p>
+            {message !== null ? (
```

Comment:
What do you think about replacing "An unexpected error occurred. Please click the button below to refresh the page." if a message is supplied?